pkg/rollout/rollout.go

```diff
@@ -285,16 +332,19 @@ func (r *Rollout) updateAnnotations(svc *run.Service, stable, candidate string)
 	if svc.Metadata.Annotations == nil {
 		svc.Metadata.Annotations = make(map[string]string)
 	}
+	now := r.time.Now().Format(time.RFC3339)
 
 	// The candidate has become the stable revision.
 	if r.promoteToStable {
 		svc.Metadata.Annotations[StableRevisionAnnotation] = candidate
 		delete(svc.Metadata.Annotations, CandidateRevisionAnnotation)
+		svc.Metadata.Annotations[LastRolloutAnnotation] = now
 		return svc
 	}
 
 	svc.Metadata.Annotations[StableRevisionAnnotation] = stable
 	svc.Metadata.Annotations[CandidateRevisionAnnotation] = candidate
+	svc.Metadata.Annotations[LastRolloutAnnotation] = now
```

Comment:
it seems like you always do this (in both cases). so why not just set it outside the `if` (e.g. right after `now:=...` ?

Reply:
Done!